tests/devapps/XamarinDev/XamarinDev/App.xaml.cs

```diff
@@ -47,7 +47,7 @@ public static string BrokerRedirectUri
         public const string DefaultAuthority = "https://login.microsoftonline.com/common";
         public const string B2cAuthority = "https://login.microsoftonline.com/tfp/msidlabb2c.onmicrosoft.com/B2C_1_SISOPolicy/";
         public const string B2CLoginAuthority = "https://msidlabb2c.b2clogin.com/tfp/msidlabb2c.onmicrosoft.com/B2C_1_SISOPolicy/";
-        public const string B2CEditProfilePolicyAuthority = "https://msidlabb2c.b2clogin.com/tfp/msidlabb2c.onmicrosoft.com/B2C_1_ProfileEditPolicy/";
+        public const string B2CEditProfilePolicyAuthority = "https://msidlabb2c.b2clogin.com/msidlabb2c.onmicrosoft.com/B2C_1_ProfileEditPolicy/";
```

Comment:
MSAL won't understand this B2C authority. The "tfp" is required. MSAL needs to understand the authority type, and this is done by parsing the URL and looking for certain substrings: 

- "adfs" for ADFS
- "dsts" for DSTS
- "tfp" for B2C